src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -3,17 +3,46 @@
 import core.basesyntax.Storage;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int ARRAY_MAX_SIZE = 10;
```

Comment:
would be nice to split static fields from non static with an empty line
```suggestion
    private static final int ARRAY_MAX_SIZE = 10;
    
```